File: app/src/main/java/com/proxy/shadowsocksr/impl/plugin/obfs/SSObfs.java
package com.proxy.shadowsocksr.impl.plugin.obfs;

public final class SSObfs extends AbsObfs
{
    public SSObfs(String usrParamStr, String rmtIP, int rmtPort, int tcpMss)
    {
        super(usrParamStr, rmtIP,rmtPort, tcpMss);
    }

    @Override public byte[] afterEncrypt(byte[] data) throws Exception
    {
        return data;
    }

    @Override public byte[] beforeDecrypt(byte[] data, boolean needsendback) throws Exception
    {
        return data;
    }
}

File: app/src/main/java/com/proxy/shadowsocksr/impl/plugin/obfs/SSObfs.kt
package com.proxy.shadowsocksr.impl.plugin.obfs

class SSObfs(usrParamStr: String, rmtIP: String, rmtPort: Int, tcpMss: Int) : AbsObfs(usrParamStr,
        rmtIP, rmtPort, tcpMss)
{
    @Throws(Exception::class)
    override fun afterEncrypt(data: ByteArray): ByteArray
    {
        return data
    }

    @Throws(Exception::class)
    override fun beforeDecrypt(data: ByteArray, needsendback: Boolean): ByteArray
    {
        return data
    }
}
